### DISCUSSION TITLE
User Welcome Message

### DISCUSSION THREAD

**Discussion by theparkerazzi** [deso-protocol/dips#5] [Medium CIPs]
I'd like to see a user welcome message that outlines the basics of how to "use" BitClout before they can ever use BitClout.com
Checking "I accept" ect..

**Comment by HPaulson** [comment]
Please detail this a bit more -- Are you thinking like a tutorial almost? Or a walkthrough when signing up?

**Reply by theparkerazzi** [reply]
Wun.vc is a great example. Shoutout BobbyDigital42

**Reply by theparkerazzi** [reply]
Look at wun.vc

**Comment by tijno** [comment]
Too much "onboarding" generally results in higher dropoff / lower retention. With notifications, distractions, commitments, family, work - we may just have 1 minute at most to wow the user before they are distracted.

Reduced friction to get the user to experience the biggest benefit of bitclout in my view is key.

On boarding can be done after that as part of getting used to the app.

Take any popular game - they dont tend to explain all the rules & moves up front - they teach you during early play. 

The hardest part is getting a user to even try bitclout.
The 2nd hardest part is to get them to come back a 2nd time.

**Reply by chafreaky** [reply]
This could be augmented by adding steps "suggested" by the app that also teach you about BitClout. For example, immediately after getting the Merlin $clout, be prompted to buy your own coin and explain the benefits of buying your coin or any coin. Gamification can go a long way.

**Reply by tijno** [reply]
+1 for gamification

**Reply by AikonMishima** [reply]
+1 with @chafreaky & initial post of @tijno.

Holding a person's hand too much will lead to them becoming quickly uninterested.

Mobile / browser instructional overlay ([examples](https://www.google.com/search?q=mobile+instructional+overlay&sxsrf=ALeKk02ng3fPhFiLjBWznn8f6EXGZ1ZmNw:1625841259623&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiCrvbVmtbxAhURC-wKHXeGAGIQ_AUoAXoECAIQAw&biw=1324&bih=843)) can help with an initial setup.

Currently working on achievement recognition as the gamification method for BitClout, which has proven successful for user adoption/retention with most community-based ecosystems.

This will, however, not impact the users' initial friction with sign-up and instantly understanding why BitClout is the place for them.

**Reply by 0xNordian** [reply]
> Mobile / browser instructional overlay (examples) can help with an initial setup.

I'm exploring this idea for the Spanish-speaking node. You can achieve it with web apps such as Toonimo or WalkMe. @AikonMishima If you are working on something like this, I would love to follow up ;)

**Reply by AikonMishima** [reply]
Toonimo and WalkMe are great!
WalkMe can be difficult to integrate though (slows down loading by a lot)

Can’t release code from my end yet as I want to make sure it’s clean and the first PoC needs to work smoothly.

General idea of the concept can be found here: lvlone.softr.app

**Reply by addanus** [reply]
Think back to the first time you used bitclout... It doesn't make sense to force a user to buy their own coin before interacting with the site. 

The way it works now is adequate - you get starter clout (usually) and can start doing social stuff. Having the user buy coin before dropping FR was a huge step forward, but we can't assume they trust the site enough to immediately buy their CC at onboarding.

Gamification is awesome - sprinkling suggestions over time (with maybe a link to explore all tips)... but really that only makes sense on dedicated apps and I wonder how much further investment is being made on bitclout.com

**Reply by arhebbar** [reply]
Love this suggestion. Instead of 10$ lumpsum. 5$ for id verification, 5$ for buying your own coin which will avoid bots buying first few coins and creating your profile id. Maybe even add a 5$ bonus for those getting 100 followers and doing 10 posts or something like that.

I see a lot of anon profiles getting the 10$ - are they even creating an id after that or cashing out.

**Comment by theparkerazzi** [comment]
Look at Wun.vc They have the right idea…

On Fri, Jul 9, 2021 at 7:08 AM chafreaky ***@***.***> wrote:

> This could be augmented by adding steps "suggested" by the app that also
> teach you about BitClout. For example, immediately after getting the Merlin
> $clout, be prompted to buy your own coin and explain the benefits of buying
> your coin or any coin. Gamification can go a long way.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bitclout/cips/discussions/5#discussioncomment-983622>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEECVASIAEUMMLMVPEFJXLTW3KDXANCNFSM5AB63BXA>
> .
>
-- 
Brandon P. Parker

757-751-7791 va
917-475-6150 ny

www.theParkerazzi.com
www.instagram.com/theparkerazzi

**Comment by theparkerazzi** [comment]
Anyone onboarding right now should only be onboarding people who want to
build the community, not participate.

On Fri, Jul 9, 2021 at 7:09 AM the Parkerazzi ***@***.***>
wrote:

> Look at Wun.vc They have the right idea…
>
> On Fri, Jul 9, 2021 at 7:08 AM chafreaky ***@***.***> wrote:
>
>> This could be augmented by adding steps "suggested" by the app that also
>> teach you about BitClout. For example, immediately after getting the Merlin
>> $clout, be prompted to buy your own coin and explain the benefits of buying
>> your coin or any coin. Gamification can go a long way.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/bitclout/cips/discussions/5#discussioncomment-983622>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AUEECVASIAEUMMLMVPEFJXLTW3KDXANCNFSM5AB63BXA>
>> .
>>
> --
> Brandon P. Parker
>
> 757-751-7791 va
> 917-475-6150 ny
>
> www.theParkerazzi.com
> www.instagram.com/theparkerazzi
>
>
>
> --
Brandon P. Parker

757-751-7791 va
917-475-6150 ny

www.theParkerazzi.com
www.instagram.com/theparkerazzi

**Comment by addanus** [comment]
I've see wun.vc mentioned a couple times... With all due respect - I believe it's a great product for the right audience. However, I found the onboarding process to be overkill. 

Like Tijn said, for someone new to the space that's not invested yet, you want to get them to experiencing the value asap.

**Comment by theparkerazzi** [comment]
It’s simply being mentioned as an idea for a roadmap. It doesn’t have to be
but a pop up or 2. Caveat Emptor and such.

On Mon, Jul 12, 2021 at 1:45 PM Addanus ***@***.***> wrote:

> I've see wun.vc mentioned a couple times... With all due respect - I
> believe it's a great product for the right audience. However, I found the
> onboarding process to be overkill.
>
> Like Tijn said, for someone new to the space that's not invested yet, you
> want to get them to experiencing the value asap.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bitclout/cips/discussions/5#discussioncomment-994990>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEECVFFIPMNIAQ5CP3OPILTXMS2XANCNFSM5AB63BXA>
> .
>
-- 
Brandon P. Parker

757-751-7791 va
917-475-6150 ny

www.theParkerazzi.com
www.instagram.com/theparkerazzi

**Reply by tijno** [reply]
fyi @theparkerazzi email replies work poorly on github it seems - as they add yours out of thread as separate threads.
